example/app/config.yml

```diff
@@ -190,10 +190,17 @@ aj:
   stackName: aj-integration
   stackNameNoDash: AjIntegration
 
-aimee-test:
-  stackName: aimee-test
-  stackNameNoDash: aimeeTest
+aimee:
+  stackName: aimee
+  stackNameNoDash: aimee
 
+  oauth:
+    provider: earthdata
+
+  api_lambda_memory: 768
```

Comment:
is there a reason that the default is 512 but yours is 768?

Reply:
It looks like ApiTokenDefault is using 495MB of memory with either token implementation and when I added any additional logging the lambda would "crash" (e.g. exceed it's memory limit), so I thought it worthwhile to make this configurable.